src/ResourceManagement/Compute/Domain/InterfaceImpl/DiskImpl.cs

```diff
@@ -432,9 +430,39 @@ Disk.Definition.IWithCreateAndSize Disk.Definition.IWithWindowsDiskSource.WithWi
         /// </summary>
         /// <param name="vhdUrl">The source VHD URL.</param>
         /// <return>The next stage of the definition.</return>
-        Disk.Definition.IWithCreateAndSize Disk.Definition.IWithWindowsDiskSource.WithWindowsFromVhd(string vhdUrl)
+        Disk.Definition.IWithStorageAccount Disk.Definition.IWithWindowsDiskSource.WithWindowsFromVhd(string vhdUrl)
         {
             return this.WithWindowsFromVhd(vhdUrl);
         }
+
+        /// <summary>
+        /// Specifies the storage account id.
+        /// </summary>
+        /// <param name="storageAccountId">The storage account id</param>
+        /// <returns>The next stage of the definition</returns>
```

Comment:
Check whether you need to add "."

Reply:
The "." is just needed in summary, not for parameters